apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/IDrugCodeProvider.java

```diff
@@ -0,0 +1,13 @@
+package gov.cms.bfd.server.war;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public interface IDrugCodeProvider {
+  String retrieveFDADrugCodeDisplay(String claimDrugCode);
+
+  Map<String, String> readFDADrugCodeFile();
+
+  Set<String> drugCodeLookupMissingFailures = new HashSet<>();
+}
```

Comment:
given the pivot away from conditional beans I suggest getting rid of the interface. either way -- these methods and the class needs documentation.